frontend/components/layout/TheMenu.vue

```diff
@@ -15,11 +15,14 @@ import {
   Play24Regular
 } from '@vicons/fluent'
 
+const config = useRuntimeConfig()
+const version = config.public.DOCUMENTATION_VERSION
+
 const links = [
   {
     id: '/documentation',
     icon: DocumentText24Regular,
-    label: 'Documentation'
+    label: `Documentation v${version}`
```

Comment:
Not sure it's a good idea to add it to the header, but I would let @LukSteib decide how it looks and if it better to move to the page itself

Reply:
waiting for @LukSteib

Reply:
if not in the header - then pls tell the spot so i don't juggle this version tag 3 times before it lands correctly.

Reply:
Also wouldn't want to add it to the header. 
Instead let's use the page itself and just add it on top of the first heading (ie. `Readme` in screenshot below) in a decent way. Doesn't have to be super prominent but good to include

<img width="450" alt="image" src="https://github.com/powerhouse-inc/switchboard-boilerplate/assets/36229155/5455310e-b61a-41ee-bc61-cccbabebd79b">

Reply:
I would suggest to keep it on the grey background right on top of the "README", to use the same font size and color as sidebar text and to say `Documentation of the package version ${version}`

Reply:
<img width="372" alt="image" src="https://github.com/powerhouse-inc/switchboard-boilerplate/assets/59374892/47487e5b-5713-4c58-9f0f-1689969eb4b8">

Reply:
> "on the grey background"

The above [suggestion](https://github.com/powerhouse-inc/switchboard-boilerplate/pull/58#discussion_r1246829380) was to not put the text on the white background to indicate that it is not part of the readme itself. If you did not follow the suggestion on purpose, please provide a reason, otherwise I assume you missed it 😞

Reply:
Ah, sorry, i misunderstood the requirement. My brain for some reason imagined that the letters are "transparent" and there's a grey background behind them 🤦🏻

Reply:
<img width="1533" alt="image" src="https://github.com/powerhouse-inc/switchboard-boilerplate/assets/59374892/0bd0c342-4fd9-4099-9b9f-d737ca25e9f8">

Reply:
Desired state

<img width="919" alt="desired-state" src="https://github.com/powerhouse-inc/switchboard-boilerplate/assets/3393626/9308f9d4-c0fd-42e7-9bd1-9aa26b60e218">

Reply:
<img width="820" alt="image" src="https://github.com/powerhouse-inc/switchboard-boilerplate/assets/59374892/56dc0995-e023-4d91-844a-8ad6e868705c">